src/vm/gcenv.ee.cpp

```diff
@@ -1440,27 +1412,7 @@ void GCToEEInterface::WalkAsyncPinned(Object* object, void* context, void (*call
     assert(object != nullptr);
```

Comment:
We can just assert false here. This method should be unreachable now.

Reply:
Apparently it's still called. The CI fails on it big time.

Reply:
Ok, I see why. Could you please change it back?

Reply:
Sure